packages/mobile/src/exchange/actions.ts

```diff
@@ -1,13 +1,17 @@
 import BigNumber from 'bignumber.js'
-import { ExchangeRatePair } from 'src/exchange/reducer'
+import { ExchangeRate, ExchangeRatePair } from 'src/exchange/reducer'
 import { CURRENCY_ENUM } from 'src/geth/consts'
 
 export enum Actions {
   FETCH_EXCHANGE_RATE = 'EXCHANGE/FETCH_EXCHANGE_RATE',
   SET_EXCHANGE_RATE = 'EXCHANGE/SET_EXCHANGE_RATE',
+  SYNC_CELO_GOLD_EXCHANGE_RATE_HISTORY = 'EXCHANGE/SYNC_CELO_GOLD_EXCHANGE_RATE_HISTORY',
+  UPDATE_CELO_GOLD_EXCHANGE_RATE_HISTORY = 'EXCHANGE/UPDATE_CELO_GOLD_EXCHANGE_RATE_HISTORY',
   EXCHANGE_TOKENS = 'EXCHANGE/EXCHANGE_TOKENS',
 }
 
+//
```

Comment:
Why all the empty comments in this file now?

Reply:
That was my attempt to separate the actions, cause it's hard to follow which is which. But that is more like a proposal, if you guys do not like it - happy to remove it :)

Reply:
I would support removing them- was also confused by the empty comments